03_launch_mgmt_cluster.sh

```diff
@@ -25,6 +25,11 @@ export GOPATH
 # BMO_RUN_LOCAL : run the baremetal operator locally (not in Kubernetes cluster)
 # CAPM3_RUN_LOCAL : run the CAPI operator locally
 
+export DEPLOY_KERNEL_URL="http://172.22.0.1:6180/images/ironic-python-agent.kernel"
+export DEPLOY_RAMDISK_URL="http://172.22.0.1:6180/images/ironic-python-agent.initramfs"
```

Comment:
Is the idea here to override the values [here](https://github.com/metal3-io/metal3-dev-env/blob/b19da74e06062d0c21ade7dea1c6e8d09f7f4e48/03_launch_mgmt_cluster.sh#L120) /to consume it in line [120-121](https://github.com/metal3-io/metal3-dev-env/blob/b19da74e06062d0c21ade7dea1c6e8d09f7f4e48/03_launch_mgmt_cluster.sh#L120)?

Reply:
No, when testing my changes in metal3-dev-env, ConfigMap content was not populated, so I had to export those variables to be consumed [here](https://github.com/metal3-io/baremetal-operator/pull/479/files#diff-758c5fb84ccf0edf8f10208948ba6717).

Reply:
@fmuyassarov that kustomization you refer to will be gone once BMO is integrated into CAPM3 deployment. Meaning that @furkatgofurov7 you need to remove the deployment of BMO, and add the deployment of Ironic when not done already

Reply:
Yes, I was confused since `launch_baremetal_operator` function was still there. 
Thanks for clarification.